<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>证书中心</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>280</height>
       </size>
      </property>
      <property name="title">
       <string>中心操作</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>240</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>文件接收</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>服务器IP</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>服务器端口</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>90</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>12346</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_6">
                   <property name="text">
                    <string>刷新IP</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton">
                   <property name="text">
                    <string>开启服务器</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>接收进度</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="progressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>根证书签名</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>签名列表</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QListWidget" name="listWidget">
                    <property name="minimumSize">
                     <size>
                      <width>300</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QGroupBox" name="groupBox_4">
                   <property name="title">
                    <string>证书签发</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <spacer name="verticalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_4">
                      <property name="maximumSize">
                       <size>
                        <width>150</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>选择文件</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="verticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <item>
                       <widget class="QLabel" name="label_4">
                        <property name="text">
                         <string>天数</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QComboBox" name="comboBox_2">
                        <property name="maximumSize">
                         <size>
                          <width>70</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="editable">
                         <bool>true</bool>
                        </property>
                        <property name="currentText">
                         <string>5</string>
                        </property>
                        <item>
                         <property name="text">
                          <string>5</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>10</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>30</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>60</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>120</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>365</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_5">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>150</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>根证书签名</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="verticalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QGroupBox" name="groupBox_5">
                   <property name="title">
                    <string>证书吊销</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <widget class="QPushButton" name="pushButton_3">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>150</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>撤销证书</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>根证书验证</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                 <item>
                  <widget class="QGroupBox" name="groupBox_10">
                   <property name="title">
                    <string>选择文件</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_15">
                    <item>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="text">
                       <string>选择证书文件</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_14">
                 <item>
                  <widget class="QGroupBox" name="groupBox_11">
                   <property name="title">
                    <string>验证证书</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_26">
                    <item>
                     <widget class="QPushButton" name="pushButton_10">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>根证书验证</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>撤销证书</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QGroupBox" name="groupBox_6">
                 <property name="title">
                  <string>撤销列表</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_14">
                  <item>
                   <widget class="QListWidget" name="listWidget_2">
                    <property name="minimumSize">
                     <size>
                      <width>300</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QGroupBox" name="groupBox_7">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="title">
                    <string>证书吊销</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <spacer name="verticalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_9">
                      <property name="text">
                       <string>生成撤销链</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="verticalSpacer_7">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_17">
                      <item>
                       <widget class="QLabel" name="label_5">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>20</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>序列号</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="lineEdit_2">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>16777215</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <spacer name="verticalSpacer_6">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_7">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>撤销证书</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="verticalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_16">
                 <item>
                  <widget class="QGroupBox" name="groupBox_8">
                   <property name="title">
                    <string>证书恢复</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_15">
                    <item>
                     <widget class="QPushButton" name="pushButton_8">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>恢复证书</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>附件预览</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QTreeView" name="treeView"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>信息显示</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>